docs/javascript/javascript-language.md

```diff
@@ -1854,4 +1854,22 @@ console.log(score);  // 0
 
 - [MDN - JavaScript First Steps - Variables](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Variables)
 - [Eloquent JavaScript - Bindings](https://eloquentjavascript.net/02_program_structure.html#h_lnOC+GBEtu)
-- [The Modern JavaScript Tutorial - Variables](https://javascript.info/variables)
\ No newline at end of file
+- [The Modern JavaScript Tutorial - Variables](https://javascript.info/variables)
+
+
+//what is needed?
+Styling DOM Nodes
+Events
+Timers
+### Styling DOM Nodes 
+
+### Events
+
+### Timers 
+Javascript has two `time intervals` methods that allows us to execute code at a given time. 
+
+The `setInterval` function will execute a given function after a waiting a certain amount of time. 
```

Comment:
Do we ever explicitly call these "callbacks" at CWHQ? That could be a good thing to enforce.

So:

```
The `setInterval()` function will execute the given callback function after...
```